src/components/SearchableMultiSelect/SearchableMultiSelect.tsx

```diff
@@ -537,9 +537,10 @@ class SearchableMultiSelect extends React.Component<
 			findTypes(props, SearchableMultiSelect.Option),
 			'props'
 		);
-		const firstVisibleIndex = _.findIndex(options, (option) => {
-			return optionFilter(searchText, option);
-		});
+		const firstVisibleIndex =
+			_.findIndex(options, (option) => {
```

Comment:
What happens if an option is not found? findIndex would return -1, does that impact this code?